docs/custom-images-contract.rst

```diff
@@ -0,0 +1,113 @@
+.. _custom-images-contract:
+
+======================
+Custom Images contract
+======================
+
+The contract is a set of requirements for crawler custom docker images to comply running with Scrapy Cloud.
+
+The contract is already implemented in Python and is known as :ref:`scrapinghub-entrypoint-scrapy <sh-entrypoint-scrapy>`. If you don’t use this package with your crawler image, you should implement Custom Images contract by yourself, or your image will not work properly in Scrapy Cloud.
```

Comment:
`scrapinghub-entrypoint-scrapy` is a *contract implementation* in python for *Scrapy projects*. It doesn't work for other python frameworks. We can probably generalise it or create another implementation for some other type like pyspider.